zcnbridge/transaction/txn.go

```diff
@@ -84,22 +86,36 @@ func (t *Transaction) ExecuteSmartContract(ctx context.Context, address, funcNam
 		return "", errors.New(errCode, msg)
 	}
 
-	t.Hash = t.scheme.GetTransactionHash()
 	if len(t.scheme.GetTransactionError()) > 0 {
 		return "", errors.New(errCode, t.scheme.GetTransactionError())
 	}
 
 	return t.Hash, nil
 }
 
-// Verify checks including of transaction in the blockchain.
-func (t *Transaction) Verify(ctx context.Context) error {
-	const errCode = "transaction_verify"
+func (t *Transaction) GetTransactionHash() string {
+	var txnout map[string]json.RawMessage
```

Comment:
The transaction hash should exist once the transaction is created and signed, we don't need to parse the transaction result to get it.

Reply:
I've removed it